fdm-app/app/entry.server.tsx

```
@@ -4,135 +4,129 @@
-            console.error(error);
-          }
-        },
-      }
-    );
+                    pipe(body)
+                },
+                onShellError(error: unknown) {
+                    reject(error)
+                },
+                onError(error: unknown) {
+                    responseStatusCode = 500
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)